src/Avalonia.X11/X11Window.cs

```diff
@@ -1164,6 +1165,27 @@ public void SetTopmost(bool value)
         public void SetEnabled(bool enable)
         {
             _disabled = !enable;
+
+            var wmHintsPtr = XGetWMHints(_x11.Display, _handle);
```

Comment:
We should be using `UpdateXxx` pattern like we do with `UpdateMotifWmHints` and `UpdateSizeHints` to keep those calculations centralized.